main.rb

```diff
@@ -28,18 +30,23 @@ def start_app
     when 6
       @show.rentals(@rentals_list)
     when 7
+      save_files
       return
     end
     main(@book_list, @people_list, @rentals_list)
   end
 end
 
 def main(book_list_input = [], people_list_input = [], rentals_list_input = [])
-  book_list = book_list_input
-  people_list = people_list_input
-  rentals_list = rentals_list_input
-  InputProcess.new(book_list, people_list, rentals_list)
+  InputProcess.new(book_list_input, people_list_input, rentals_list_input)
 end
 
-main
-# This is outrageous Microverse. Thanks.
+books = File.open('data/books.json', 'r+')
+people = File.open('data/people.json', 'r+')
+rentals = File.open('data/rental.json', 'r+')
```

Comment:
- [ ] You are saving values to `rentals.json` but trying to read values from `rental.json`. Please be sure you are saving and reading values from the same file.